src/PrestaShopBundle/Bridge/Smarty/ToolbarFlagsConfigurator.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Bridge\Smarty;
+
+use Language;
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShopBundle\Bridge\AdminController\ControllerConfiguration;
+use Symfony\Component\Routing\RouterInterface;
+use Tools;
+
+/**
+ * This class sets help link, maintenance information, title, and others in the controller configuration.
+ */
+class ToolbarFlagsConfigurator implements ConfiguratorInterface
+{
+    //todo This url must be replace after split
+    private const HELP_URL = 'https://help.prestashop.com/';
+
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * @param RouterInterface $router
+     * @param Configuration $configuration
+     */
+    public function __construct(RouterInterface $router, Configuration $configuration)
+    {
+        $this->router = $router;
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * @param ControllerConfiguration $controllerConfiguration
+     *
+     * @return void
+     */
+    public function configure(ControllerConfiguration $controllerConfiguration): void
+    {
+        $this->initToolbar($controllerConfiguration);
+        $this->initPageHeaderToolbar($controllerConfiguration);
+
+        $controllerConfiguration->templatesVars['maintenance_mode'] = !(bool) $this->configuration->get('PS_SHOP_ENABLE');
+        $controllerConfiguration->templatesVars['debug_mode'] = (bool) _PS_MODE_DEV_;
+        $controllerConfiguration->templatesVars['lite_display'] = $controllerConfiguration->liteDisplay;
+        $controllerConfiguration->templatesVars['url_post'] = $this->router->generate('admin_features_index');
```

Comment:
Hardcoded URL

Reply:
I will do this in the next PR, and I will create an issue to note all your remark